accounts-journey/src/androidTest/java/screens/components/AccountComponent.kt

```diff
@@ -0,0 +1,40 @@
+package screens.components
+
+import android.view.View
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.hasSibling
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import app_common.shouldBeDisplayed
+import app_common.shouldContainText
+import com.backbase.accounts_journey.R
+import org.hamcrest.CoreMatchers.allOf
+import org.hamcrest.Matcher
+
+/**
+ * Class that provides references to the ViewInteraction objects for each view in the list items
+ * Requires that the item has a unique name.
+ */
+class AccountComponent(accountName: String) {
+
+    private val accountNameMatcher: Matcher<View> = allOf(withId(R.id.account_name), ViewMatchers.withText(accountName))
```

Comment:
Could add an import for `ViewMatchers.withText` to use `withText` instead of `ViewMatchers.withText`?

Reply:
Sure, done